README.md

```diff
@@ -1,3 +1,103 @@
-# Docker Traceability Plugin
+# CloudBees Docker Traceability
 
-This plugin allows the tracking of the creation and use of Docker containers in Jenkins.
+This [Jenkins](http://jenkins-ci.org) plugin allows the tracking of the creation and use of Docker containers in Jenkins and their future use.
+
+#Plugin Summary
+
+* Container deployments summary page
+* Tracking of Docker image and container deployments produced in Jenkins
+* Tracking of Docker container events
+* Tracking of Docker container states being retrieved from *docker inspect* calls
+* Advanced [API](#API):
+ * Submission of events from Docker and Docker Swarm installations
+ * Data polling from Jenkins (including Docker API-compatible JSONs)
+ * Support of search queries
+ 
+ ![Main page](/doc/images/root-action.png)
+
+#Installation Guidelines
+
+##Plugin setup
+
+1. Install CloudBees Docker Traceability plugin from Jenkins Update Center
+2. Install other Jenkins plugins, which produce image fingerprints to be traced by the plugin (see [Integrations][#Integrations])
```

Comment:
I use Atom with a Markdown preview, but the link seemed to be correct there. BTW it violates the format spec in any case